src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionTestResultHeaderView.swift

```diff
@@ -98,27 +91,31 @@ class ExposureSubmissionTestResultHeaderView: DynamicTableViewHeaderFooterView {
 		column.centerYAnchor.constraint(equalTo: baseView.centerYAnchor).isActive = true
 		column.leftAnchor.constraint(equalTo: barView.rightAnchor, constant: 25).isActive = true
 
-		subTitleLabel = UILabel()
-		subTitleLabel.text = "subTitle"
-		subTitleLabel.font = UIFont.systemFont(ofSize: 13)
+		subTitleLabel = ENALabel()
+		subTitleLabel.text = "Footnote"
+		subTitleLabel.numberOfLines = 0
+		subTitleLabel.style = .footnote
+		subTitleLabel.textColor = .enaColor(for: .textPrimary2)
 		subTitleLabel.translatesAutoresizingMaskIntoConstraints = false
 		column.addSubview(subTitleLabel)
 		subTitleLabel.leftAnchor.constraint(equalTo: column.leftAnchor, constant: 5).isActive = true
 		subTitleLabel.topAnchor.constraint(equalTo: barView.topAnchor).isActive = true
 
-		titleLabel = UILabel()
-		titleLabel.text = "title"
+		titleLabel = ENALabel()
+		titleLabel.text = "Title 2"
```

Comment:
Do we have/need an AppStrings entry for this?

Reply:
Not required, as they are just placeholders. Will be overridden when the header is configured anyway.

Reply:
👍